xbmc/video/windows/VideoFileItemListModifier.cpp

```diff
@@ -85,11 +85,23 @@ void CVideoFileItemListModifier::AddQueuingFolder(CFileItemList& items)
     // @note: the items list contains the (..) upper directory navigation fileitem plus all the
     // season directory fileitems for a given show. We want to assign the "All Seasons" listitem
     // the infotag of the tv show - so do not use the first item in the list!
-    if (items.Size() && items[items.Size() - 1]->GetVideoInfoTag())
+    // instead we need to make sure to get a season item, so loop through the list for the case
```

Comment:
I was unaware of this feature, this makes it more robust and should work in all cases.
Can you adjust the comment too? The first item in the list may not be the (..) up directory (user might have that option disabled) and after your change it doesn't really matter. We just have to make sure the item is a season.
Thanks for the fix

Reply:
if i got it right the first item is always (..) regardless of the setting. this is why the loop starts with `int i = 1` not `int i = 0`. i'd check it and adjust the comment accordingly

Reply:
my bad, sure (..) depends on the setting. i adjusted the comment